relayer/src/cache.rs

```diff
@@ -15,6 +15,10 @@ const CONNECTION_CACHE_TTL: Duration = Duration::from_secs(10 * 60);
 const CLIENT_STATE_CACHE_TTL: Duration = Duration::from_millis(500);
 const LATEST_HEIGHT_CACHE_TTL: Duration = Duration::from_millis(200);
 
+const CHANNEL_CACHE_CAPACITY: u64 = 10_000;
```

Comment:
This could be further tuned to a number that the relayer could be expected to actively maintain.